utils/detect_sound.py

```
@@ -1,42 +1,43 @@
-import sys
 import os
+import platform
 import subprocess
 
+
 def get_process_names():
     """Return a set of running process names."""
     try:
-        # pgrep -a . returns full command lines
         output = subprocess.check_output(["pgrep", "-a", "."], text=True).splitlines()
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)